libs/libusb/DETAILS

```diff
@@ -1,12 +1,12 @@
           MODULE=libusb
-         VERSION=1.0.23
+         VERSION=1.0.24
           SOURCE=$MODULE-$VERSION.tar.bz2
 SOURCE_DIRECTORY=$BUILD_DIRECTORY/$MODULE-$VERSION
       SOURCE_URL=http://github.com/libusb/libusb/releases/download/v$VERSION/
```

Comment:
Perhaps changing the source url to;

SOURCE_URL_FULL=https://github.com/libusb/libusb/archive/v$VERSION.tar.gz

would fix the problem